app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,12 @@
+
+child :content_facet => :content_attributes do
+  extends 'katello/api/v2/content_facet/base'
+
+  node do |content_facet|
+    version = content_facet.content_view_version
+    {
+      :content_view_version => version.version,
+      :content_view_version_id => version.id
```

Comment:
Why do we want version information only in the show data? Versus the base which includes the content view ID and name.